tests/flashcore/test_rating_system_inconsistency.py

```
@@ -0,0 +1,333 @@
+        # Create review manager
+        manager = ReviewSessionManager(
+            db_manager=in_memory_db,
+            scheduler=FSRS_Scheduler(),
+            user_uuid=uuid4(),
+            deck_name=sample_card.deck_name
+        )
+        manager.initialize_session()
+
+        # Submit review with unified rating (1-4)
+        rating = 3  # Good
+        updated_card = manager.submit_review(
```

Ruff (0.11.9)

Local variable `updated_card` is assigned to but never used

Remove assignment to unused variable `updated_card`

(F841)